app/views/versions/show.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="page-header">
```

Comment:
Oh. I did write that into the page as you can see in the screenshot, just forgot to push the commit :(